packages/backend-common/src/service/lib/ServiceBuilderImpl.ts

```diff
@@ -169,7 +182,14 @@ export class ServiceBuilderImpl implements ServiceBuilder {
       app.use(root, route);
     }
     app.use(notFoundHandler());
-    app.use(errorHandler());
+
+    if (this.errorHandler) {
+      app.use(this.errorHandler);
+    }
+
+    if (this.useDefaultErrorHandler) {
```

Comment:
Can two error handlers be used at the same time(default and custom)? Otherwise add this code to an else statement above

Reply:
Yep, we can decide if we want just custom, just default or both(useful when we want to add sentry integration)

Reply:
Yep, we can decide if we want just custom, just default or both(useful when we want to add sentry integration)

Reply:
That makes a lot of sense. Doing error reporting in a custom handler and then passing it further to the defaultErrorHandler for logging and constructing the json error response